scenariolib/visit.go

```diff
@@ -134,11 +134,16 @@ func (v *Visit) ExecuteScenario(scenario Scenario, c *Config) error {
 		if v.WaitBetweenActions {
 			var timeToWait int
 			if c.TimeBetweenActions > 0 {
-				timeToWait = c.TimeBetweenActions
+				timeToWait = c.TimeBetweenActions //Random waiting time
+				WaitBetweenActions(timeToWait)
+			}else if c.TimeBetweenActions < 0 {
+				timeToWait = c.TimeBetweenActions*(-1)
+				time.Sleep(time.Duration(timeToWait) * time.Second) //Constant Waiting time
```

Comment:
Why not remove the "random" from `WaitBetweenActions`, it is not clear it randomizes the time anyway.

This way, you could have:

```
 if c.TimeBetweenActions > 0 {
    timeToWait = rand.Intn(c.TimeBetweenActions)
} else if c.TimeBetweenActions < 0 {
    timeToWait = c.TimeBetweenActions * (-1)
} else {
    timeToWait = rand.Intn(DEFAULTTIMEBETWEENACTIONS)
}
WaitBetweenActions(timeToWait)
```

No need for comments since the code is clearer.